src/encoded/schemas/software.json

```diff
@@ -80,21 +80,70 @@
                 "title": "Purpose",
                 "type": "string",
                 "enum": [
+                    "3' RACE",
```

Comment:
This is an unfortunate consequence...these are esther's changes but now they will be overwritten as Bek's.